integrations/huggingface-transformers/tutorials/sparse-transfer-learning-bert.md

```diff
@@ -0,0 +1,343 @@
+- `QuantizationModifier` instructs SparseML to apply quantization aware training to quantize the weights over the final epochs
+- `DistillationModifier` instructs SparseML to apply model distillation at the logit layer
+   
+As a result, we end up with a 95% pruned and quantized BERT trained on SST2! It achieves ~92% accuracy.
+
+#### Aside: Dense Teacher Creation (OPTIONAL)
+
+By passing a `--distill_teacher` argument, we instruct SparseML to apply model distillation during the fine-tuning process. This is optional, but passing a dense teacher can help improve accuracy.
+
+Above, we instructed SparseML to download the dense teacher from SparseZoo. You can also pass a path to a directory containing a your transformers model and configuration.
+
+Here is the command we used to train the dense teacher:
+
+```bash
+sparseml.transformers.text_classification \
+  --output_dir dense_obert-text_classification_sst2 \
+  --model_name_or_path zoo:nlp/masked_language_modeling/obert-base/pytorch/huggingface/wikipedia_bookcorpus/base-none \
+  --recipe zoo:nlp/sentiment_analysis/obert-base/pytorch/huggingface/sst2/base-none \
+  --task_name sst2 --max_seq_length 128 --per_device_train_batch_size 32 --per_device_eval_batch_size 32 --preprocessing_num_workers 6 \
+  --do_train --do_eval --evaluation_strategy epoch --fp16 --seed 20811 \
+  --save_strategy epoch --save_total_limit 1
+```
+
+Note that the SparseZoo stubs passed to `--recipe` and `--model_name_or_path` here both end in `base-none`. This identifies the standard dense versions of the models/recipes in [SparseZoo](zoo:nlp/masked_language_modeling/obert-base/pytorch/huggingface/wikipedia_bookcorpus/base-none).
+
+Here is what the `recipe` looks like:
+
+```yaml
+version: 1.1.0
+
+# General Variables
+num_epochs: 2
+init_lr: 2e-5 
+final_lr: 0
+
+# Modifiers:
+training_modifiers:
+  - !EpochRangeModifier
+      end_epoch: eval(num_epochs)
+      start_epoch: 0.0
+    
+  - !LearningRateFunctionModifier
+    start_epoch: 0
+    end_epoch: eval(num_epochs)
+    lr_func: linear
+    init_lr: eval(init_lr)
+    final_lr: eval(final_lr)
+```
+
+As you can see, this recipe only has modifiers for the learning rate schedule and number of epochs. As such, during the training
+process, the model remains dense SparseML is not instructed to apply any pruning or quantization algorithms.
+
+The dense teacher achieves ~92.9% validation accuracy.
+
+### Exporting to ONNX
+
+The SparseML installation provides a `sparseml.transformers.export_onnx` command that you can use to export the model to ONNX. Be sure the `--model_path` argument points to your trained model:
+
+```bash
+sparseml.transformers.export_onnx \
+    --model_path ./pruned_quantized_obert-text_classification_sst2 \
+    --task text_classification
+```
+
+The command creates a `./deployment` folder in your local directory, which contains the ONNX file and necessary Hugging Face tokenizer and configuration files.
```

Comment:
The command creates a ./deployment folder in your local directory containing the ONNX file and necessary Hugging Face tokenizer and configuration files.